src/components/comment-input.tsx

```diff
@@ -20,11 +22,31 @@ const InputContainer = styled.div`
 `;
 
 export const CommentInput: React.VoidFunctionComponent = () => {
+  const { setNewCommentText, newCommentText, comments, setComments } =
+    useComments();
+
+  const onSend = () => {
```

Comment:
When we define a local event handler inside a component, we can choose to call it `handleSomething` instead of `onSomething`. This helps distinguish local variables from props inside jsx.